packages/playwright/src/index.ts

```diff
@@ -185,8 +185,16 @@ export default class AxeBuilder {
 
   private async inject(frames: Frame[]): Promise<void> {
     for (const iframe of frames) {
-      await iframe.evaluate(await this.script());
-      await iframe.evaluate(await this.axeConfigure());
+      const race = new Promise((_, reject) => {
```

Comment:
Playwright does not have a pageload timeout like all the other drivers, and neither does the `evaluate` function have a timeout option. So this was the only other choice to have the lazy loaded iframe not hang indefinitely